docs/variants/msi_z690/test-matrix.md

```diff
@@ -68,7 +68,7 @@ Certification Procedure for both tested platforms:
 | No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
 |:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
 | 1.   | [TPM Support][TPM]                                | TPM           | Without TPM001.001 and TPM002.001    |
-| 2.   | [Verified Boot support][VBO]                      | VBO           | VBO006.002, VBO007.002, VBO008.001   |
+| 2.   | [Verified Boot support][VBO]                      | VBO           | All                                  |
```

Comment:
It should be `Without VBO006.001 and VBO007.001` according to spreadsheets? And `VBO009.001` and `VBO010.001` should be added to spreadsheet if I'm not mistaken.